schema.graphql

```diff
@@ -23,3 +23,41 @@ type Count @entity {
   id: ID!
   value: BigInt!
 }
+
+enum NonceUpdateType @entity {
+  CONTRACT
+  SIGNER
+  ASSET
+}
+
+type NoncesUpdateHistory @entity {
+  id: ID!
+  type: NonceUpdateType!
+  date: BigInt!
+  sender: String!
+  contractUpdate: NoncesUpdateContractHistory
+  singerUpdate: NoncesUpdateSingerHistory
+  assetUpdate: NoncesUpdateAssetHistory
+}
+
+type NoncesUpdateContractHistory @entity {
+  id: ID!
+  fromNonce: BigInt!
+  toNonce: BigInt!
+}
+
+type NoncesUpdateSingerHistory @entity {
+  id: ID!
+  fromNonce: BigInt!
+  toNonce: BigInt!
+  signer: String!
+}
+
+type NoncesUpdateAssetHistory @entity {
+  id: ID!
+  fromNonce: BigInt!
```

Comment:
The event will stop sending both the previous and current nonce. It wass irrelevant as nonces can be increased by one everytime, you can keep this but might be unnecessary.

Reply:
This can be changed afterwards when the new contract is deployed. But lets keep it in mind